features/guidance/turn-angles.feature

```diff
@@ -0,0 +1,1104 @@
+                                                                        k
+            """
+
+        And the ways
+            | nodes  | name  | oneway | highway   | lanes |
+            | ab     | rose  | yes    | secondary | 2     |
+            | ef     | rose  | yes    | secondary | 2     |
+            | de     | trift | yes    | secondary | 2     |
+            | bc     | trift | yes    | secondary | 2     |
+            | eghibj | muhle | yes    | tertiary  | 1     |
+
+        And the nodes
+            | node | highway         |
+            | h    | traffic_signals |
+            | j    | traffic_signals |
+
+        When I route I should get
+            | waypoints | route             | turns                              |
+            | a,c       | rose,trift,trift  | depart,new name slight left,arrive |
+            | a,k       | rose,muhle,muhle  | depart,turn slight right,arrive    |
+            | d,f       | trift,rose,rose   | depart,new name straight,arrive    |
+            | d,k       | trift,muhle,muhle | depart,turn sharp left,arrive      |
+            | d,c       | trift,trift,trift | depart,continue uturn,arrive       |
+
+    #http://www.openstreetmap.org/#map=19/52.50740/13.44824
+    Scenario: Turning Loop at the end of the road
+        Given the node map
+            """
+                                                      l - - k  _
+                                                    .             j
+                                                  m                 '
+                                                 .                    i
+                                                .                      .
+                                                .                       .
+                                                .                       h
+                                                .                       .
+                                                n                       .
+                                                .                       .
+                                                .                       g
+                                                o                       .
+                                               .                       .
+                                              p                       f
+                                           .                         .
+                                       .                            .
+                                   .                            . e
+            a - - - - b - - - - c - - - - - - - - - - - - - d '
+            """
+
+     And the ways
+            | nodes             | name    | highway     | lanes |
+            | abcdefghijklmnopc | circled | residential | 1     |
+
+     When I route I should get
+            | waypoints | bearings     | route           | turns         |
+            | b,a       | 90,2 180,180 | circled,circled | depart,arrive |
```

Comment:
Oh I see, 90,2 at b forces going straight with a wiggle room of 2 degree. Thanks!